PokeOneWeb/Frontend/pokeoneweb/src/app/pages/pokemon-detail/pokemon-detail.component.ts

```diff
@@ -46,14 +46,17 @@ export class PokemonDetailComponent implements OnInit {
             this.pokemonService.getByNameFull(this.model.pokemonName).subscribe((result) => {
                 this.model.pokemon = result as IPokemonVarietyModel;
 
-                this.titleService.setTitle(`${this.model.pokemon.name} - ${WEBSITE_NAME}`);
-
-                this.model.learnableMoves = this.model.pokemon.learnableMoves;
-
-                this.sortMoveLearnMethods();
-                this.sortForms();
-                this.applyInitialSorting();
-            });
+                    this.titleService.setTitle(`${this.model.pokemon.name} - ${WEBSITE_NAME}`);
+
+                    this.model.learnableMoves = this.model.pokemon.learnableMoves;
+                    
+                    if(this.model.areEventExclusiveSpawnsHidden) {
```

Comment:
if-statement is redundant, as event-exclusive spawns are always hidden on page load.

Reply:
What if the default behavior would be changed in PokemonDetailModel?
Wouldn't you then have to change it manually in the init-function?